Server/src/main/java/local/AssetDAOImpl.java

```diff
@@ -232,6 +232,17 @@ public void insertAsset(Asset asset) {
         }
     }
 
+    public void updateRecommendation(int assetID, String recommendation) {
+        try {
+            PreparedStatement ps = getConnection().prepareStatement(UPDATE_RECOMMENDATION);
```

Comment:
SonarQube comment. It is better to surround the prepared statements with a try-with-resources block. This is hinted at by IntelliJ as well. It makes sure that the resources are properly closed after usage. See other methods in the class.

Reply:
Resolved in the last commit.